test/robot/trace/client.go

```diff
@@ -17,16 +17,14 @@ package trace
 import (
 	"context"
 	"fmt"
-	"sync"
 	"time"
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/google/gapid/core/app/crash"
 	"github.com/google/gapid/core/app/layout"
 	"github.com/google/gapid/core/log"
-	"github.com/google/gapid/core/os/android/adb"
+	device "github.com/google/gapid/core/os/device"
```

Comment:
`device` is already the package name. Drop the explicit naming.

Reply:
Done